compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/ConstantValueResolver.java

```diff
@@ -215,6 +218,26 @@ private BLangConstantValue calculateConstValue(BLangConstantValue lhs, BLangCons
         return new BLangConstantValue(null, this.currentConstSymbol.type);
     }
 
+    private BLangConstantValue calculateConstValue(BLangConstantValue value, OperatorKind kind) {
```

Comment:
This method is only called for unary expressions right, shall we change  the name of the method to reflect that, something like `evaluateUnaryOperator` or something. In fact I don't see any value addition by having this bit of logic as a separate method. I think we can just inline this method.

Reply:
Yes, it only calls for unary expressions. Will change the method name to reflect that